lib/src/controls/better_player_material_controls.dart

```diff
@@ -807,12 +809,25 @@ class _BetterPlayerMaterialControlsState
   Widget _buildExitButton() => GestureDetector(
         behavior: HitTestBehavior.translucent,
         onTap: () {
-          _betterPlayerController!
-              .postEvent(BetterPlayerEvent(BetterPlayerEventType.close));
+          if (!_betterPlayerController!.isFullScreen) {
+            Navigator.of(context).pop();
```

Comment:
tutaj raczej bym dał funkcje ktora przekazać i wywolac, onExitButtonPressed i jak jest nullable to dopiero zrobic pop

Reply:
U nas w apce w customControllerze jest tak funkcja ale wywoływana jest z nasze apki na podstawie te eventu `postEvent(BetterPlayerEvent(BetterPlayerEventType.close))` tutaj dałem tego popa bo jak nie mamy full screena to wychodzi nam z playera do ekranu vod z karuzelami a ma wrócić do details